packaging/test_irods_client_rest_cpp.py

```diff
@@ -276,6 +276,27 @@ def test_list(self):
                 shutil.rmtree(dir_name)
                 admin.assert_icommand(['irm', '-f', '-r', dir_name])
 
+    def test_list_honors_recursive__issue114():
```

Comment:
Should this accept a `self` for the `unittest` method(s)? I think this will raise an Exception on 294 as it is.

Reply:
Done